plugins/hls-cabal-plugin/src/Ide/Plugin/Cabal/Completion/Data.hs

```diff
@@ -181,6 +181,7 @@ libExecTestBenchCommons =
       ("ghcjs-prof-options:", constantCompleter ghcOptions),
       ("ghcjs-shared-options:", constantCompleter ghcOptions),
       ("includes:", filePathCompleter),
+      ("autogen-includes:", filePathCompleter),
```

Comment:
It would be better to add the `autogen-modules` completer in this spot as well, so that we don't get the duplicated code for each stanza.